isengard.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
```

Comment:
From the name of this file. I really cannot tell what this file does. Is this a Karma configuration file? Why is it name isengard.conf.js?